src/EditorFeatures/Core/Implementation/LanguageServer/AlwaysActivateInProcLanguageClient.cs

```diff
@@ -52,7 +52,10 @@ internal class AlwaysActivateInProcLanguageClient : AbstractInProcLanguageClient
 
         public override ServerCapabilities GetCapabilities(ClientCapabilities clientCapabilities)
         {
-            var serverCapabilities = new VSInternalServerCapabilities();
+            var serverCapabilities = new VSInternalServerCapabilities
```

Comment:
ah this will need to be moved down - if the lsp editor is enabled it overrides this value.

Reply:
gotcha.